java/src/jmri/implementation/configurexml/MatrixSignalMastXml.java

```diff
@@ -30,7 +30,7 @@ public MatrixSignalMastXml() {
     public Element store(Object o) { // from mast p to XML
         MatrixSignalMast p = (MatrixSignalMast) o;
 
-        Element e = new Element("signalmast");
+        Element e = new Element("matrixsignalmast");
```

Comment:
I'm not sure that I fully agree with the need to put this in to a new schema version. Yes we are adding a new element but, in the way it's done, it's not mandated and files generated with the content of the current version should still be valid with this new element in place.